google-cloud-tools-plugin/src/com/google/gct/idea/appengine/cloud/ManagedVmCloudType.java

```diff
@@ -161,11 +160,27 @@ public ManagedVmServerConnector(
 
     @Override
     public void connect(@NotNull ConnectionCallback<ManagedVmDeploymentConfiguration> callback) {
-      callback.connected(new ManagedVmRuntimeInstance(configuration));
+      try {
+        getFileFromFilePath(configuration.getCloudSdkExecutablePath());
```

Comment:
Just calling this method with this name might not convey immediately that it is used for validation purpose only here. Could it be wrapped into a method with name like `ensureSdkExecutableExists()`, return a boolean and handle the exception there?